Calling/public/index.html

```diff
@@ -1,17 +1,38 @@
 ﻿<!-- Copyright (c) Microsoft Corporation. -->
 <!-- Licensed under the MIT License. -->
 
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Azure Communication Services - UI Library: Calling Sample app" />
     <title>UI Library Calling Sample</title>
+
```

Comment:
please remove this as its for render tracker a debugging tool we don't provide in the hero